File: templates/java/fr/text/code/invertLine.java
import stanford.karel.*;
public class InverJetons extends Karel {
   /** commencer l'exécution ici */
   public void run() {
      while(devantDegage()) {
         invertiJeton();
         avancer();
      }
      /* pour empêcher un bug fencepost */
      invertiJeton();
   }

   /*  prend un jeton si on est présent pose un jeton sin
    * on */
   public void invertiJeton() {
      /* une déclaration if / else */
      if(jetonsPresents()) {
         ramasserJeton();
      } else {
         laisserJeton();
      }
   }
}

File: templates/python/fr/text/code/invertLine.py
from karel.stanfordkarel import *

# le début du programme
def main():
   while l'avant_est_clair(): 
      inverser_jeton()
      avancer()
   # pour éviter un bug de clôture
   inverser_jeton()

# prend un jeton s'il y en a un
# pose un jeton sinon
def inverser_jeton():
   # une instruction if / else
   if jetons_présent():
      ramasser_jeton()
   else:
      laisser_jeton()